benches/dual_numbers.rs

```diff
@@ -49,7 +49,7 @@ fn bench_dual_numbers<E: EquationOfState>(
         b.iter(|| a_res((&state.eos, &state.derive1(Derivative::DV))))
     });
     group.bench_function("a_hyperdual", |b| {
-        b.iter(|| a_res((&state.eos, &state.derive2(Derivative::DV, Derivative::DV))))
+        b.iter(|| a_res((&state.eos, &state.derive2_mixed(Derivative::DV, Derivative::DV))))
```

Comment:
any chance that the compiler outsmarts us here and optimizes that somehow? Is the performance the same for DVDT?

Reply:
I'll add a test case for that.